src/main/resources/avro/bdg.avdl

```diff
@@ -748,166 +748,187 @@ record Variant {
-   scores are separated by whether or not the base supports the reference or the
-   alternate allele.
+   Genotype posterior probabilities in the range 0.0 to 1.0 using the same ordering as the
+   genotype likelihood field; one use can be to store imputed genotype probabilities. VCF genotype
+   field reserved key GP, Number=G.
    */
-  union { null, float } baseQRankSum = null;
+  array<float> posteriorProbabilities = [];
 
+  // GQ Number=1 Type=Integer
   /**
-   The Fisher's exact test score for the strand bias of the reference and alternate
-   alleles. Stored as a phred scaled probability. Thus, if:
-
-   * a = The number of positive strand reads covering the reference allele
-   * b = The number of positive strand reads covering the alternate allele
-   * c = The number of negative strand reads covering the reference allele
-   * d = The number of negative strand reads covering the alternate allele
-
-   This value takes the score:
-   
-   -10 log((a + b)! * (c + d)! * (a + c)! * (b + d)! / (a! b! c! d! n!)
-
-   Where n = a + b + c + d.
+   Conditional genotype quality, encoded as a phred quality −10log10 p(genotype call is wrong
+   conditioned on the site being variant). VCF genotype field reserved key GQ.
    */
-  union { null, float } fisherStrandBiasPValue = null;
+  union { null, int } quality = null;
 
+  // MQ Number=1 Type=Integer
   /**
-   The root mean square of the mapping qualities of reads covering this site.
+   Root Mean Square (RMS) of the mapping quality of reads for this genotype. VCF genotype field
+   reserved key MQ.
    */
-  union { null, float } rmsMapQ = null;
-  
-  /**
-   The number of reads at this site with mapping quality equal to 0.
-   */
-  union { null, int } mapq0Reads = null;
+  union { null, int } rmsMappingQuality = null;
 
+  // PQ Number=1 Type=Integer
   /**
-   The Wilcoxon rank-sum test statistic of the mapping quality scores. The mapping
-   quality scores are separated by whether or not the read supported the reference or the
-   alternate allele.
+   Phred-scaled probability that alleles are ordered incorrectly, against all other members in
+   the same phase set. VCF genotype field reserved key PQ.
    */
-  union { null, float } mqRankSum = null;
+  union { null, int } phaseSetQuality = null;
 
-  /**
-   The Wilcoxon rank-sum test statistic of the position of the base in the read at this site.
-   The positions are separated by whether or not the base supports the reference or the
-   alternate allele.
-   */
-  union { null, float } readPositionRankSum = null;
+  //
+  // Not present in VCF specification yet present in Genotype
 
   /**
-   The log scale prior probabilities of the various genotype states at this site.
-   The number of elements in this array should be equal to the ploidy at this
-   site, plus 1.
+   Minimum number of reads seen at this site across samples when joint calling variants.
+   VCF genotype field key MIN_DP.
```

Comment:
Is it that simple?

"[the minimum]...at this site across all samples" doesn't mean the same thing as "[the minimum]...across a set of non-variant sites grouped together into non-variant block records that represent intervals of sites for which the genotype quality (GQ) is within a certain range or band."  I don't believe we support
```
##GVCFBlock=minGQ=0(inclusive),maxGQ=5(exclusive)
##GVCFBlock=minGQ=20(inclusive),maxGQ=60(exclusive)
##GVCFBlock=minGQ=5(inclusive),maxGQ=20(exclusive)
##GVCFBlock=minGQ=60(inclusive),maxGQ=2147483647(exclusive)
```
header stuff like HaplotypeCaller outputs unless those are passed through unintentionally.

Or am I reading to much into the current description; it meant gVCF blocks all along?